synthtool/gcp/templates/python_library/.kokoro/docs/docs-presubmit.cfg

```diff
@@ -15,3 +15,14 @@ env_vars: {
     key: "TRAMPOLINE_IMAGE_UPLOAD"
     value: "false"
 }
+
+env_vars: {
+    key: "TRAMPOLINE_BUILD_FILE"
+    value: "github/python-bigquery/.kokoro/build.sh"
```

Comment:
Surely this needs to be templated if a change to the general config

Reply:
Oops definitely, thank you for catching this.